tests/datadriven/test_variants.py

```diff
@@ -56,11 +57,77 @@ def getDataModelClass(self):
     def getProtocolClass(self):
         return protocol.GAVariantSet
 
-    def verifyVariantsEqual(self, gaVariants, pyvcfVariants):
+    def _compareTwoFloats(self, a, b):
```

Comment:
Yeah this should be assertAlmostEqual or something, placed in the test superclass... check the python unittest documentation for the signature